utils/precompileSvgs/precompilationDiff.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Print gif diff of publicly exposed Vue icons
+ * (defined in 'lib/KIcon/precompiled-icons/iconDefinitions.js')
+ * to allow for easier before/after precompilation preview
+ */
+
+const fs = require('fs');
+const path = require('path');
+const { exec } = require('child_process');
+const consola = require('consola');
+
+const ICONS_DEFINITIONS_PATH = './lib/KIcon/iconDefinitions.js';
+
+// Within the main icon definitions object, match all individual definition objects:
+// `someIconName: { icon: require('./some/icon/path'), ...}`
+// (taken from extractRstIcons.js)
+const ICON_DEFINITION_PATTERN = /\n {2}(\w+): {.*?\s+icon:\s+require\('(.+?)'\).*?},/gs;
+
+function print() {
+  fs.readFile(ICONS_DEFINITIONS_PATH, 'utf8', (error, iconsDefinitions) => {
+    if (error) {
+      consola.error(`Can't open icons definitions file: "${ICONS_DEFINITIONS_PATH}"`);
+      return;
+    }
+
+    let iconDefinitionMatch;
+    while ((iconDefinitionMatch = ICON_DEFINITION_PATTERN.exec(iconsDefinitions)) !== null) {
+      const iconAlias = iconDefinitionMatch[1];
+      const iconPath = path.resolve(path.join('./lib/KIcon/', iconDefinitionMatch[2]));
+
+      exec('git diff ' + iconPath, (error, stdout, stderr) => {
```

Comment:
```suggestion
      exec('git diff --color=always ' + iconPath, (error, stdout, stderr) => {
```

This will preserve the red/green colored diff output:

<img width="1351" alt="image" src="https://user-images.githubusercontent.com/2367265/124151558-abdc7700-da60-11eb-823c-d3ca3845789f.png">

Reply:
Wonderful, thanks